toolchain/check/context.cpp

```diff
@@ -1368,6 +1367,17 @@ auto Context::GetInterfaceType(SemIR::InterfaceId interface_id,
       *this, FacetTypeFromInterface(interface_id, specific_id).facet_type_id);
 }
 
+auto Context::GetInt32Type() -> SemIR::TypeId {
+  auto bit_width_const_id = TryEvalInst(
+      *this, SemIR::InstId::Invalid,
+      SemIR::IntValue{
+          .type_id = GetBuiltinType(SemIR::BuiltinInstKind::IntLiteralType),
+          .int_id = ints().AddUnsigned(llvm::APInt(64, 32))});
```

Comment:
```suggestion
          .int_id = ints().Add(32)});
```

I believe the end result here is going to be equivalent (the unsigned value gets canonicalized to a signed value), but this is probably cheaper since it avoids a whole APInt round trip that's not needed due to the small value.